src/index.html

```diff
@@ -17,61 +17,63 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="#" class="nav__link">Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Смарфоны и телефоны</a>
+            <a href="#" class="nav__link">Смарфоны и телефоны</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Ноутбуки и компьютеры</a>
+            <a href="#" class="nav__link">Ноутбуки и компьютеры</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Гаджеты</a>
+            <a href="#" class="nav__link">Гаджеты</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Планшеты</a>
+            <a href="#" class="nav__link">Планшеты</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Фото</a>
+            <a href="#" class="nav__link">Фото</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Видео</a>
+            <a href="#" class="nav__link">Видео</a>
```

Comment:
```suggestion
            <a href="#" class="nav__link">
              Видео
            </a>
```

Запропонував би перенести, як дівки)